components/aero_sensors/FS/fatfs.c

```diff
@@ -0,0 +1,633 @@
+/**
+ * @file fatfs.h
+ * @brief  Source code for the FatFS interface
+ * @author Joshua Lafleur (josh.lafleur@outlook.com)
+ * @version 0.1
+ * @date 2022-07-16
```

Comment:
```suggestion
 * @file fatfs.c
 * @brief  Source code for the FatFS interface
```

Please don't put authors, versions, dates, or anything else that will have to be updated continuously in any code.